src/components/AboutContainer/AboutContentContainer/AboutContentContainer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import useStyles from './AboutContentContainer.styles';
+import { WhatWeDo, WhoWeAre, Programs } from './AboutContentPages';
+import { PropTypes } from 'prop-types';
+
+const AboutContentContainer = ({ selected }) => {
+  const { container, content } = useStyles();
+
+  const page = (() => {
+    let result;
+    switch (selected) {
+      case 'What We Do':
+        result = <WhatWeDo />;
+        break;
+      case 'Who We Are':
+        result = <WhoWeAre />;
+        break;
+      case 'Programs':
+        result = <Programs />;
+        break;
+      default:
+        <WhatWeDo />;
+    }
+    return result;
+  })();
+
+  return (
+    <section className={container}>
+      <article className={content}>{page}</article>
```

Comment:
invoke the func

`<article className={content}>{page()}</article>`

Reply:
`page` is an IIFE so it is already invoked. Do you think IIFE is ok in this case?

Reply:
or should I stick to regular function to avoid confusion

Reply:
personally, not really a fan of IIFE, unless there is a need to (just my stylistic preference i guess?); for example, you're other scenario of IIFE in the WhoWeAre component is acceptable cause you're passing it as a props and using the object's name as the prop's name (line 57 and 58)...
![image](https://user-images.githubusercontent.com/42846522/114805938-e42dbc80-9d71-11eb-9936-12b79b5f1edc.png)

Reply:
Fixed in 8fec142ba0870f95295740ed59241bb935ef2872